account_export_csv/wizard/account_export_csv_view.xml

```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="account_csv_export_view" model="ir.ui.view">
+         <field name="name">Accounting Entries CSV Export</field>
+         <field name="model">account.csv.export</field>
+         <field name="arch" type="xml">
+            <form string="Accounting CSV Export">
+                <group>
+                    <group>
+                        <field name="company_id" />
+                    </group>
+                 </group>
+                <group>
+                    <group>
+                        <field name="date_range_id"  domain="['|', ('company_id', '=', company_id), ('company_id', '=', False)]"/>
+                        <label for="date_start" string="Date" class="oe_inline"/>
+                        <div>
+                            <field name="date_start" class="oe_inline"/>
+                            <span string="-" class="oe_inline"/>
+                            <field name="date_end" class="oe_inline"/>
+                        </div>
+                    </group>
+                    <group>
+                    </group>
+                </group>
+                <div>
+                    <label for="journal_ids" />
+                    <field name="journal_ids"/>
+                </div>
+               <separator string ="Report" colspan="4"/>
+                <group colspan="4">
+                    <field name="data" filename="export_filename"/>
+                </group>
+                <footer>
+                    <button name="action_manual_export_account" string="Trial Balance" type="object" icon="fa-cog" class="oe_highlight"/>
+                    <button name="action_manual_export_analytic" string="Analytic Balance (with accounts)" type="object" icon="fa-cog" class="oe_highlight"/>
+                    <button name="action_manual_export_journal_entries" string="Journal Entries" type="object" icon="fa-cog" class="oe_highlight"/>
```

Comment:
oe_highlight class should be removed on every button.
too many highlight leads to confusion